dependencies/android/README.md

```diff
@@ -1,25 +1,23 @@
-# ANDROID SETUP
+# Android
+
+## Install dependencies
+
+* libogg: [Download](https://xiph.org/downloads/) and unzip it in `dependencies/android/libogg`.
+
+* libtheora: [Download](https://xiph.org/downloads/) and unzip it in `dependencies/android/libtheora`.
 
-* Clone the repo, then install the dependencies listed below
 * Ensure the symbolic links in `[root]/android/app/jni` are correct: 
   * `RSDKv5` -> root of the RSDKv5 repository
   * `Game` -> root of the Mania repository (or any other game that you may be compiling)
   * Extra symbolic links can be added for things like mods, **as mods do not use the local files for logic.** Just ensure that there is an `Android.mk` file at the root of them. 
-    * (to add symbolic links, use the following:)
-      * Windows: `mklink /D "name-of-symlink" "[path]"`
-      * Linux: `ln -s "name-of-symlink" "[path]"`
+  
+    To add symbolic links, do the following:
+      * Windows: `mklink /d "[name-of-symlink]" "[path]"`
+      * Linux: `ln -s "[path]" "[name-of-symlink]"`
 * Open `[root]/android/` in Android Studio, install the NDK and everything else that it asks for, and build.
 
-The working directory will be at `[sdcard root]/RSDK/v5`. **Please note that GL shaders are required, or you will get a black screen.** See [the shaders README](../../RSDKv5/Shaders/README.md) for more details.
+The working directory will be at `[sdcard root]/RSDK/v5`. **Please note that GL shaders are required, or you will get a black screen.** Read the Shaders section in [the GL3 README](../../RSDKv5/Shaders/README.md) for a guide on how to install them.
```

Comment:
States to see the shaders section in the GL3 readme, but still points to the shaders readme.

Reply:
wait i can do this myself i think